daisy_workflows/image_build/enterprise_linux/ks_helpers.py

```diff
@@ -260,14 +260,21 @@ def BuildKsConfig(release, google_cloud_repo, byos, sap, uefi):
     custom_post = '\n'.join([ol_post, el_post])
     cleanup = FetchConfigPart('el7-cleanup.cfg')
     repo_version = 'el7'
-  elif release == "rhel8":
+  elif release == "rhel8" or release.startswith('rhel-8'):
     logging.info('Building RHEL 8 image.')
     ks_packages = FetchConfigPart('el8-packages.cfg')
     ks_options = FetchConfigPart('el8-options.cfg')
     if uefi:
       logging.info('Building RHEL 8 for UEFI')
       ks_options = FetchConfigPart('el8-uefi-options.cfg')
     rhel_post = FetchConfigPart('rhel8-post.cfg')
+    if sap:
+      logging.info('Building RHEL 8 for SAP')
+      point = ''
```

Comment:
maybe a nitpick but if you're just appending, you could just call this `rhel_post`